recipes/osx-pocl-opencl/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: osx-pocl-opencl
+  version: 1
+
+build:
+  number: 0
+  track_features:
+    - osx_pocl_opencl
+  skip: True  # [not osx]
+
+test: {}
+
+about:
+  home: http://github.com/conda-forge/osx-pocl-opencl-feedstock
+  license: BSD 3-clause
+  license_family: BSD
+  summary: Metapackage to track whether OpenCL on OSX is being provided by POCL (rather than the OS)
+
+extra:
+  recipe-maintainers:
+    - inducer
+    - mattwala
```

Comment:
@mattwala is one of my grad students. I've removed him for now--I can add him back later if he likes.